libs/styles/consonant-play-button.css

```diff
@@ -0,0 +1,115 @@
+
+ .modal-img-link {
+  position: relative;
+  display: inline-block;
+  height: 100%;
+  z-index: 4;
+}
+
+.modal-img-link .consonant-play-btn {
+  position: absolute;
+  height: 77px;
+  width: 77px;
+  left: calc(50% - 38.5px);
+  top: calc(50% - 38.5px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: var(--link-color-dark);
+  box-sizing: border-box;
+  border: 2.5px solid var(--color-white);
+  border-radius: 50%;
+  outline: none;
+}
+
+.modal-img-link .consonant-play-btn.btn-small {
+  height: 44px;
+  width: 44px;
+  left: calc(50% - 22px);
+  top: calc(50% - 22px);
+  border: 2px solid var(--color-white);
+}
+
+.modal-img-link .consonant-play-btn:hover {
+  background-color: var(--color-accent-down);
+}
+
+.modal-img-link .consonant-play-btn:focus,
+.modal-img-link .consonant-play-btn:focus-visible {
+  outline: 2.5px solid var(--link-color-dark);
+  outline-offset: 0;
+  background-color: var(--color-accent-down);
+}
+
+.modal-img-link .consonant-play-btn.btn-small:focus,
+.modal-img-link .consonant-play-btn.btn-small:focus-visible {
+  outline: 2px solid var(--link-color-dark);
+}
+
+.modal-img-link .consonant-play-btn:active,
+.modal-img-link .consonant-play-btn.btn-small:active {
+  outline: none;
+  background-color: var(--color-accent-down);
+}
+
+.modal-img-link .consonant-play-btn .play-icon-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  width: 32px;
+  height: 32px;
+}
+
+[dir="rtl"] .modal-img-link .consonant-play-btn .play-icon-container {
+  align-items: flex-start;
+}
+
+.modal-img-link .consonant-play-btn.btn-small .play-icon-container {
+  width: 18px;
+  height: 18px;
+}
+
+.modal-img-link .consonant-play-btn .play-icon {
+  width: 24px;
+  height: 32px;
+  flex-shrink: 0;
+  fill: var(--color-white);
+}
+
+.modal-img-link .consonant-play-btn.btn-small .play-icon {
+  width: 13.5px;
+  height: 18px;
+}
+
+@media screen and (min-width: 600px) {
+  .modal-img-link .consonant-play-btn.btn-large {
+    height: 102px;
+    width: 102px;
+    left: calc(50% - 51px);
+    top: calc(50% - 51px);
+    box-sizing: border-box;
+    border: 3px solid var(--color-white);
+  }
+
+  .modal-img-link .consonant-play-btn.btn-large:focus,
+  .modal-img-link .consonant-play-btn.btn-large:focus-visible {
+    outline: 3px solid var(--link-color-dark);
```

Comment:
```suggestion
    outline-width: 3px;
```

Reply:
@ryanmparrish I have made the suggested changes.